mne/viz/ica.py

```diff
@@ -28,7 +28,8 @@
 @fill_doc
 def plot_ica_sources(ica, inst, picks=None, start=None,
                      stop=None, title=None, show=True, block=False,
-                     show_first_samp=False, show_scrollbars=True):
+                     show_first_samp=False, show_real_time=False,
+                     show_scrollbars=True):
```

Comment:
Same comment regarding parameter order, the new parameter should probably be appended to the end.